signalfx-org-metrics/metrics.yaml

```diff
@@ -378,6 +378,46 @@ sf.org.datapointsTotalCountByToken:
         * Data resolution: 10 seconds
   metric_type: counter
   title: sf.org.datapointsTotalCountByToken  
+ 
+sf.org.datainventoryAdded:
+  brief: DEF
+  description: |
+    DEF
+    
+        * Dimension(s): `orgId`, ?
```

Comment:
Same dimensions for all of them
- type: eq. `AWS`
- service
- region
- integrationId

Reply:
Added!